api/grpcserver.go

```diff
@@ -273,11 +273,27 @@ func (svr *GRPCServer) ReadState(ctx context.Context, in *iotexapi.ReadStateRequ
 
 // EstimateGasForAction estimates gas for action
 func (svr *GRPCServer) EstimateGasForAction(ctx context.Context, in *iotexapi.EstimateGasForActionRequest) (*iotexapi.EstimateGasForActionResponse, error) {
```

Comment:
what is the difference between `EstimateGasForAction` and `EstimateActionGasConsumption`?

Reply:
EstimateActionGasConsumption distinguishes action, and the result is more accurate.